Source/Core/VideoCore.cpp

```diff
@@ -264,4 +282,16 @@ const struct per_item VideoPerItem [Item_VideoMax]=
     { Group_IDET_R,    Group_VideoMax,     "bottom",        "lavfi.idet.repeated.bottom", 2,  false,   DBL_MAX, DBL_MAX },
     { Group_IDET_R,    Group_VideoMax,     "top",           "lavfi.idet.repeated.top",    2,  false,   DBL_MAX, DBL_MAX },
     { Group_IDET_R,    Group_VideoMax,     "neither",       "lavfi.idet.repeated.neither",     2,  false,   DBL_MAX, DBL_MAX },
+    // pkt_duration_time & pkt_size
+    { Group_Time_And_Size,    Group_VideoMax,     "pkt_duration_time",        "lavfi.signalstats.PKTDUR", 2,  false,   DBL_MAX, DBL_MAX },
```

Comment:
Why "lavfi.signalstats.PKTDUR" rather than "pkt_duration_time"? I'm confused what PKTDUR relates to.